### DISCUSSION TITLE
plot trajectory

### DISCUSSION THREAD

**Discussion by vdkxmx** [OpenDrift/opendrift#1276] [Q&A]
Hello,
I have a question about plot. When I plot particles trajectories use o.plot, there are trajectories of all particles and all timesteps, but I just want plot some trajectories of specific particles or some specific time interrvals, are there some methods?

**Comment by knutfrode** [comment]
Hi,
For the most flexibility, I would recommend trying the new TrajnAn package:
https://opendrift.github.io/trajan/gallery/example_opendrift.html#sphx-glr-gallery-example-opendrift-py

In the near future, OpenDrift will user TrajAn instead of internal methods for plotting and analysis.

However, it is also possible to sub-select trajectories and time-period when importing with OpenDrift:
`>>> o = opendrift.open(<netCDF-file>, times=times, elements=elements)`
where `times` and `elements` are the indices of the timesteps and elements that you want to import.
